docs/src/add-services/redis.md

```diff
@@ -127,11 +121,12 @@ highlight=python
 
 ## Multiple databases
 
-Redis 3.0 and above are configured to support up to 64 databases.
-Redis doesn't support distinct users for different databases
-so one relationship connection gives access to all databases.
+Redis 3.0 and above support up to 64 databases.
+But you can't set up different access rights to each database.
+When you set up a relationship connection,
+access to all the databases is automatically granted.
 
-The way to access a particular database depends on the [client library](https://redis.io/clients) you're using:
+The way you can access a particular database depends on the [client library](https://redis.io/clients) you're using:
```

Comment:
:question: Can I ask why you made this change? It feels less natural to me and slightly more confusing as there's an extra modal verb. But I very well could be missing something.